### DISCUSSION TITLE
About useSupported computed properties

### DISCUSSION THREAD

**Discussion by Alfred-Skyblue** [vueuse/vueuse#3473] [General]
In `useSupported`, a computed value is returned, which is often used to determine whether the browser supports a particular API. However, whether a browser supports a specific API is determined at the time of browser rendering and does not change during runtime due to some reasons. Therefore, in many cases, it should not be a computed property. Should we consider adding an option to return it as a regular value instead of a computed one?

For instance, `useMutationObserver` is determined at the browser's initialization whether it supports `MutationObserver`

https://github.com/vueuse/vueuse/blob/aca6a79227f7f8b06760d756dfbb4cc5958b445d/packages/core/useMutationObserver/index.ts#L27

**Comment by darrylnoakes** [comment]
This was added for SSR support, because certain features are not supported when server rendering, but are on the client, and the value needs to be able to change.

See the PR (#1800) and commit (81d92252) for more info.

Would whatever you are thinking of fit into this?
